txgithub/api.py

```diff
@@ -40,11 +40,14 @@ class GithubApi(object):
     # - optional user/pass auth (token is not available with v3)
     # - async API
 
-    def __init__(self, oauth2_token, baseURL=None):
+    def __init__(self, oauth2_token, baseURL=None, _reactor=None):
```

Comment:
Underscores don't make arguments private, since they can still be passed prepositionally. In any case, it is sensible to allow the reactor to be specified.